### PR TITLE
HW 2. GIT, Workflow, Sequences

### PR DESCRIPTION
**Pull request by albina-pasichnyk**
In scope of this pull request following tasks were resolved:
Task 1 - Companies list after overtake
Task 2 - Blacklists intersection
Task 3 - Guests food preferences
Task 4 - Guests in rooms
Task 5 - Processing unique names
Each task was committed separately with proper commit message.

### CODE REVIEW

File: Companies.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Defining 2 Companies Eleks and Toshiba
+Eleks = ['Albina', 'Kostiantyn', 'Bohdan', 'James', 'Sam', 'Peter']
+Toshiba = ['Halyna', 'Oksana', 'Paul', 'Peter', 'Soul', 'Bohdan', 'Katalin']
+
+# Company Toshiba takes over Eleks company
+Toshiba.extend(Eleks)
```
Comment (bohdandovbsyh) [inline Companies.py:6]: it is not completed)) after this manipulation eleks still has employees. Please add one line of code to fix it

File: Unique_names.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+print(f"List of non-unique names: {non_unique_names}")
+
+# Copy list of non-unique names to another variable, to keep it original
+processed_names = non_unique_names.copy()
+
+# Generate list of people with unique names
+for each_name in processed_names:
+    name_counter = processed_names.count(each_name)
+    if name_counter != 1:
+        processed_names.remove(each_name)
```
Comment (bohdandovbsyh) [inline Unique_names.py]: It is correct, but as I mentioned in the lesson, you should use default functions of different types. Think about one which can help here.

File: Companies.py (hunk 1 above)
Comment (albina-pasichnyk) [reply Companies.py:6]: Issue was resolved by adding clear function for Eleks list

File: Unique_names.py (hunk 2 above)
Comment (albina-pasichnyk) [reply Unique_names.py]: Issue was resolved by using dictionary as temporary variable and its default functions (.fromkeys and .keys).

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**